AOBJ/type/zif_aff_aobj_v1.intf.abap

```diff
@@ -0,0 +1,74 @@
+INTERFACE zif_aff_aobj_v1
+
+  PUBLIC.
+
+  TYPES: ty_write_class TYPE c LENGTH 40.
+  TYPES: ty_delete_class TYPE c LENGTH 40.
+  TYPES: ty_reload_class TYPE c LENGTH 40.
+
+
+  TYPES:
+    BEGIN OF ty_attributes,
+      "! <p class="shorttext">Name of executable class which writes the data to the archive</p>
+      "! Name of executable class which writes the data to the archive
+      "! $required
+      write_class  TYPE ty_write_class,
+      "! <p class="shorttext">Name of executable class which deletes the data from the tables</p>
+      "! Name of executable class which deletes the data from the tables
+      "! $required
+      delete_class TYPE ty_delete_class,
+      "! <p class="shorttext">Name of executable class which reloads the data to the tables</p>
+      "! Name of executable class which reloads the data to the the tables (may be empty)
+      reload_class TYPE ty_reload_class,
+    END OF ty_attributes.
+
+  TYPES:
+    "! <p class="shorttext">Table from which data is archived</p>
+    "! Tables of business object instance from which data is archived
+    BEGIN OF ty_table_to_be_archived,
+      "! <p class="shorttext">Name of table or structure</p>
```

Comment:
In the description, you only mention tables from which data is archived. Are structures correct here?
If not, I would suggest to use 
```suggestion
      "! <p class="shorttext">Name</p>
```

Reply:
Changed as suggested